src/main/resources/assets/computercraft/lua/rom/programs/shell.lua

```diff
@@ -194,6 +194,10 @@ function shell.setDir( _sDir )
     if not fs.isDir( _sDir ) then
         error( "Not a directory", 2 )
     end
+    local sStartChar = string.sub( _sDir, 1, 1 )
```

Comment:
Can we use `_sDir:sub(1, 1)` here too? Just so we're consistent with what's below.